websites/I/I Love Music/presence.ts

```diff
@@ -1,228 +1,118 @@
-//______________________________________________________________________________________
 const presence = new Presence({
-	clientId: "477919120789078026",
-});
-let { language } = navigator;
-
-switch (language) {
-	// By ACertainCoder#9011
-	//German
-	//---------------------------------------
-	case "de":
-	case "de-CH":
-	case "de-AT":
-	case "de-LU":
-	case "de-LI":
-		language = "de";
-		break;
-
-	//English / Unknown
-	//---------------------------------------
-	case "en":
-	case "en-US":
-	case "en-EG":
-	case "en-AU":
-	case "en-GB":
-	case "en-CA":
-	case "en-NZ":
-	case "en-IE":
-	case "en-ZA":
-	case "en-JM":
-	case "en-BZ":
-	case "en-TT":
-	default:
-		language = "en";
-		break;
+		clientId: "477919120789078026",
+	}),
+	browsingStamp = Math.floor(Date.now() / 1000);
+async function getStrings() {
+	return presence.getStrings(
+		{
+			browse: "general.browsing",
+			buttonViewPage: "general.buttonViewPage",
+			search: "general.searchFor",
+		},
+		await presence.getSetting<string>("lang").catch(() => "de")
+	);
+}
+let strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
+enum Assets {
+	Logo = "https://i.imgur.com/jO86Aqk.png",
+	Play = "https://i.imgur.com/OLaz6JN.png",
+	Search = "https://i.imgur.com/oGQtnIY.png",
 }
-//__________________________________________________________________________________________
 
+async function imgPath(path: string, hostname: string) {
+	if (path) {
+		if (path.includes(hostname)) return `https://${path.replace("//", "")}`;
+		else return `https://${hostname}${path}`;
+	} else return Assets.Logo;
+}
+function capitalizeFirstLetter(string: string) {
+	if (string) {
```

Comment:
Why is this check needed?

Reply:
it isn’t

Reply:
Remove it then

Reply:
okay on it

Reply:
It is needed cuz it may be passed undefined values.